runtime/doc/diagnostic.txt

```diff
@@ -888,6 +888,30 @@ open_float({opts})                               *vim.diagnostic.open_float()*
         (`integer?`) float_bufnr
         (`integer?`) winid
 
+                                       *vim.diagnostic.render_virtual_lines()*
+render_virtual_lines({namespace}, {bufnr}, {diagnostics})
+    Displays the given diagnostics as virtual lines.
```

Comment:
I plan on including an example of how to use this function with `on_jump` after merging https://github.com/neovim/neovim/pull/33850.